workers/generic-worker/loopback_video_linux.go

```diff
@@ -0,0 +1,46 @@
+//go:build linux
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/taskcluster/taskcluster/v53/workers/generic-worker/host"
+)
+
+func setupVideoDevice(task *TaskRun) *CommandExecutionError {
+	err := host.Run("/usr/sbin/modprobe", "v4l2loopback", fmt.Sprintf("video_nr=%d", config.LoopbackVideoDeviceNumber))
+	if err != nil {
+		return executionError(internalError, errored, fmt.Errorf("Could not load the v4l2loopback kernel module: %v", err))
+	}
+
+	devicePath := fmt.Sprintf("/dev/video%d", config.LoopbackVideoDeviceNumber)
```

Comment:
I feel like devicePath should be a struct member of LoopbackVideoTask

Reply:
Good idea, thanks.